src/main/java/Duke.java

```diff
@@ -1,10 +1,197 @@
+import duke.Deadline;
+import duke.Event;
+import duke.Task;
+import duke.ToDo;
+
+import java.util.Scanner;
+import java.util.Arrays;
+import java.util.ArrayList;
+
 public class Duke {
+
+    public static final String BYE = "bye";
+
     public static void main(String[] args) {
-        String logo = " ____        _        \n"
-                + "|  _ \\ _   _| | _____ \n"
-                + "| | | | | | | |/ / _ \\\n"
-                + "| |_| | |_| |   <  __/\n"
-                + "|____/ \\__,_|_|\\_\\___|\n";
-        System.out.println("Hello from\n" + logo);
+        printWelcomeMessage();
+        ArrayList<Task> taskLists = new ArrayList<Task>();
+
+
+        String userInput = getUserInput();
+
+        while(!userInput.equals("bye")){
+            String[] userInputSplit = userInput.split(" ");
+            switch(userInputSplit[0]){
+            case "list":
+                printTaskList(taskLists, Task.numOfTasks);
+                break;
+
+            case "mark":
+                int markDoneIndex = Integer.parseInt(userInputSplit[1]) - 1;
+                taskLists.get(markDoneIndex).setIsDone(true);
+                printSetDoneMessage(taskLists.get(markDoneIndex));
+                break;
+
+            case "todo":
+                try{
+                    addTodo(taskLists, userInputSplit);
+                }
+                catch(EmptyDescriptionException e){
+                    System.out.println(e);
+                }
+
+                break;
+
+            case "event":
+                try{
+                    addEvent(taskLists, userInputSplit);
+                }
+                catch(StringIndexOutOfBoundsException e){
+                    System.out.println(e.toString());
+                };
+                break;
+
+            case "deadline":
+                addDeadline(taskLists, userInputSplit);
+                break;
+
+
+            case "unmark":
+                int markNotDoneIndex = Integer.parseInt(userInputSplit[1]) - 1;
+                taskLists.get(markNotDoneIndex).setIsDone(false);
+                printSetNotDoneMessage(taskLists.get(markNotDoneIndex));
+                break;
+
+            case "delete":
+                int deleteIndex = Integer.parseInt(userInputSplit[1]) - 1;
+                deleteTask(taskLists, deleteIndex);
+                break;
+
+                default:
```

Comment:
Default should have the same indent level as your other cases in `switch`, as per the Java coding standard.